{{ cookiecutter.project_slug }}/api/pyproject.toml

```diff
@@ -0,0 +1,102 @@
+[tool.pytest.ini_options]
+DJANGO_SETTINGS_MODULE = "{{ cookiecutter.project_slug }}.settings"
+python_files = ["tests.py", "test_*.py", "*_test.py"]
+
+[tool.black]
+line-length = 120
+target-version = ['py310']
+exclude = '''
+/(
+    \.eggs
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.env
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+  | node_modules
+)/
+'''
+
+[tool.bandit]
+skips = ['B101',]
+
+[tool.isort]
+profile = 'black'
+line_length=120
+multi_line_output = 3
```

Comment:
This option looks redundant, because 3 is default value for black profile https://pycqa.github.io/isort/docs/configuration/profiles.html

Reply:
❤️